packages/autocomplete-theme-classic/src/theme.scss

```
@@ -1,229 +1,412 @@
+                color: var(--text-color);
+                // remove native appearence
+                &::-webkit-search-decoration,
+                &::-webkit-search-cancel-button,
+                &::-webkit-search-results-button,
+                &::-webkit-search-results-decoration {
+                    -webkit-appearance: none;
+                }
+                &::placeholder {
+                    opacity: 1;
+                    color: var(--muted-color);
+                }
+                // remove focus effect as we moved it to parent wrapper
+                &:focus {
+                    border-color: none;
+                    box-shadow: none;
+                    outline: none;
+                }
+            }
+        }
+        .aa-InputWrapperSuffix {
+            order: 4;
+            display: flex;
+            align-items: center;
+            // accelerator to clear the query
+            button.aa-ResetButton {
+                display: flex;
+                align-items: center;
+                border: 0;
+                cursor: pointer;
+                background: none;
+                color: var(--muted-color);
+                &[hidden] {
+                    display: none;
+                }
+                &:hover {
+                    color: var(--text-color);
+                }
+                svg {
+                    width: var(--icon-size);
+                    stroke-width: var(--icon-stroke-width);
+                }
+            }
+        }
+    }
 }
 
-.aa-TouchFormContainer {
-  display: flex;
-  grid-gap: 0.5rem;
-  justify-content: space-between;
+// ----------------
+// Panel
+// ----------------
+.aa-Panel {
+    position: absolute;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    .aa-PanelLayout {
+        //display: flex;
+        margin-top: var(--spacing-half);
+        padding-top: var(--spacing-half);
+        border-radius: 3px;
+        background-color: var(--background-color);
+        box-shadow: var(--panel-shadow);
+        text-align: left;
+        padding-bottom: var(--spacing-half);
+        overflow: hidden;
+        .col-results {
+            // max-height: 400px;
+            width: 50%;
+            overflow-y: scroll;
+            // section.aa-Source
+        }
+        aside.col-preview {
+            //max-height: 400px;
+            //animation: fadeIn ease-in .2s;
+            flex-shrink: 1;
+            max-width: 50%;
+            overflow: hidden;
+            border-left: solid 1px var(--selected-color);
+            &:empty {
+                //display:none;
+            }
+        }
+    }
 }
 
-.aa-TouchCancelButton {
-  background: none;
-  border: 0;
-  color: inherit;
-  cursor: pointer;
-  font-size: inherit;
-  padding: 0;
+// when a query isn't resolved yet
+.aa-Panel--stalled {
```

```suggestion
&--stalled {
```

could be this, when it's nested in aa-Panel